L/LibPMC/build_tarballs.jl

```diff
@@ -0,0 +1,33 @@
+using BinaryBuilder
+
+name = "LibPMC"
```

Comment:
Quoting from the [documentation section](https://github.com/JuliaPackaging/Yggdrasil/pull/6026#discussion_r1056998837) I [suggested reading](https://github.com/JuliaPackaging/Yggdrasil/pull/6026#discussion_r1056998837):

>  It should be the name of the upstream package, not for example that of a specific library or executable provided by it

Looks like a more appropriate name would be PMC?

Reply:
ok sure, I've changed it.